### ISSUE TITLE
Some features does not working

### ISSUE THREAD

**Issue by atayahmet** [vlio20/ng2-date-picker#21]
Hi @vlio20 

First, thank you for this awesome package.

Some features i want to use do not work. Used and does not work features below.

## Configuration:

| Name  |  Type | Default  | Description  |  
|---|---|---|---|
| closeOnSelect  |  Boolean |  true |  If set to true will close the date-picker after choosing a date from the calender, otherwise, won't | 

## Attributes:

| Name  |  Type | Default  | Description  |  
|---|---|---|---|
| minDate|  Moment or String|  undefined| This is a validation rule, if the selected date will be before minDate the containing form will be invalid. Note: if provided as string format configuration should be provided in the config object | 

I also could not work events.

**For example:** 

| Name  |  Type   | Description  |  
|---|---|---|
| dayClick|  EventEmitter<Moment[]>|  Emits the currently selected days every time the selection changes| 

**Use Case:**

```html
<dp-day-picker
    [(ngModel)]="install"
     name="install"
     #install="ngModel"
     [required]="true"
     [class.dp-material]="true"
     [config]="datePickerConfig"
     [minDate]="'2017-05-16'"
     (dayClick)="catchEvent($event)">
</dp-day-picker>
```

**Main config:**

```javascript
public datePickerConfig: Object = {
    format: "DD/MM/YYYY",
    closeOnSelect: true,
    weekdayNames: {
      "su": "Pz",
      "mo": "Pzt",
      "tu": "Sal",
      "we": "Çar",
      "th": "Per",
      "fr": "Cum",
      "sa": "Cmt"
    }
  };

```
Thanks.

**Comment by atayahmet** [comment]
Hi again,

Please ignore the issue about attribute. Smoothly works.

**Comment by vlio20** [comment]
@atayahmet, thanks for opening a bug. Will take care of it soon.

**Comment by vlio20** [comment]
@atayahmet, issue was solved and available in 0.5.0 version (there was also small api change `dayContextmenu` -> `dayContextMenu`) .
